### PR TITLE
Updated translations from Phrase

### PR DESCRIPTION
**Pull request by terraware-phrase**
Exported the most recent locale files from Phrase

### CODE REVIEW

File: src/strings/csv/es.csv [hunk 1]
```diff
@@ -1131,16 +1199,27 @@ RECEIVED_DATE,Fecha de recepción,
...
 REGION,Región,
+REGION_ANTARCTICA,Antártida,
+REGION_EAST_ASIA_PACIFIC,Asia Oriental y Pacífico,
+REGION_EUROPE_CENTRAL_ASIA,Europa y Asia Central,
+REGION_LATIN_AMERICA_CARIBBEAN,América Latina y el Caribe,
+REGION_MIDDLE_EAST_NORTH_AFRICA,Oriente Medio y África del Norte,
+REGION_NORTH_AMERICA,América del Norte,
+REGION_OCEANIA,Oceanía,
+REGION_SOUTH_ASIA,Asia del Sur,
+REGION_SUB_SAHARAN_AFRICA,África subsahariana,
```
Comment (karthikbtf) [inline src/strings/csv/es.csv:1222]: Some of these don't match BE translations.
```
src/main/resources/i18n/Enums_es.properties:public.Region.Antarctica=La Antártida
src/main/resources/i18n/Enums_es.properties:public.Region.EastAsiaPacific=Asia Oriental y el Pacífico
src/main/resources/i18n/Enums_es.properties:public.Region.LatinAmericaCaribbean=Latinoamérica y el Caribe
src/main/resources/i18n/Enums_es.properties:public.Region.NorthAmerica=Norteamérica
src/main/resources/i18n/Enums_es.properties:public.Region.SouthAsia=Sudeste Asiático

REGION_ANTARCTICA,Antártida,
REGION_EAST_ASIA_PACIFIC,Asia Oriental y Pacífico,
REGION_LATIN_AMERICA_CARIBBEAN,América Latina y el Caribe,
REGION_NORTH_AMERICA,América del Norte,
REGION_SOUTH_ASIA,Asia del Sur,
```

We may need to update these in `phrase`.

**Review by karthikbtf — APPROVED** [review]

File: src/strings/csv/es.csv (hunk 1 above)
Comment (karthikbtf) [reply src/strings/csv/es.csv:1226]: I don't have access to Phrase, would you be able to edit the values @nickgraz ?

Comment (nickgraz) [reply src/strings/csv/es.csv:1226]: I have access, but just noticed I can't see the projects. I will fix when I get access.

Comment (karthikbtf) [reply src/strings/csv/es.csv:1226]: Thanks @nickgraz sounds good.

**Pull request merged**